librz/include/rz_core.h

```diff
@@ -418,6 +418,20 @@ RZ_API void rz_core_item_free(RzCoreItem *ci);
 
 typedef int (*RzCoreSearchCallback)(RzCore *core, ut64 from, ut8 *buf, int len);
 
+typedef struct sym_name_t {
+	const char *pfx; // prefix for flags
+	char *name; // raw symbol name
+	char *rz_symbol_name;
+	char *libname; // name of the lib this symbol is specific to, if any
+	char *nameflag; // flag name for symbol
+	char *demname; // demangled raw symbol name
+	char *demflag; // flag name for demangled symbol
+	char *classname; // classname
+	char *classflag; // flag for classname
+	char *methname; // methods [class]::[method]
+	char *methflag; // methods flag sym.[class].[method]
+} SymName;
```

Comment:
This should have a doxygen description, moreover any exported structure should start with `Rz`.